docs/turnstile-migration.md

```diff
@@ -2,40 +2,27 @@
-* Uses round numbers to obscure any identifying information as these round number values, e.g. 10 will be very common in the blockchain.
+Please see the [zcashd blog post](https://electriccoin.co/blog/sprout-to-sapling-migration-tool/) for further details.
 
-## Using the turnstile tool
+!!! tip "Customising the destination Sapling address."
+    By default, the destination address will be the default Sapling address of the wallet. You may specify a different Sapling address to migrate to by specifying the `migrationdestaddress` parameter in your [`zcash.conf` file](/using-zecwallet/#customising-zcashconf).
 
-![Turnstile migration](images/turnstile-start.png)
+To check on the progress of the turnstile at any time, visit **Apps->Sapling Turnstile** and you will be presented with the full migration history detailing all transactions that have occurred as well as the amount left to migrate.
 
-To start the turnstile migration tool, open it by choosing **Apps->Sapling Turnstile** and choose the Sprout address (zc) to migrate funds from and the Sapling address (zs) to send funds too in addition to the number of days you would like to migrate the funds over.
-
-Generally, increasing the option to migrate over a larger number of blocks is the preferred solution but note that there will be an increase in the number of transactions, all of which require a fee. The migrate over 4 days (2304 blocks) requires 10 transactions resulting in fees of 0.001 ZEC for the migration.
-
-Note that you may only perform one migration at a time using the app.
-
-!!! tip "Closing the wallet during the migration."
-    If you shut down the wallet, then it'll send any pending transactions on reopening, which may result in the undesired behaviour of multiple transactions occurring back-to-back. The wallet **must** be open for any transactions to be sent so you should open it frequently and preferably leave it open throughout.
-
-To check on the progress of the turnstile at any time, visit **Apps->Sapling Turnstile** and you will be presented with the migration progress and the number of blocks until the next transaction.
-
-![Turnstile progress](images/turnstile-progress.png)
-
-!!! danger "Backup your wallet.dat file after starting the migration."
-    ZecWallet creates **new** transparent addresses for the turnstile migration. You must create a backup after confirming the migration to ensure you have the private keys for these intermediate transparent addresses in case there is an issue. To do so, visit **File->Backup wallet.dat**.
+!!! info "Closing the wallet during the migration."
+    If you shut down the wallet, then no funds will be migrated. The wallet will start to prepare transactions 5 blocks before the 500 block height interval (to allow for the time to create Sprout transactions) so the wallet should be open during this period. If the wallet is not open and a 500 block height interval passes, no action will take place until the next 500 block height interval.
```

Comment:
Should we mention that if you are running an external zcashd, then it will keep doing the migration even if zecwallet is closed?